packages/flutter/lib/src/rendering/object.dart

```diff
@@ -912,7 +1022,8 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
           debugPrintStack();
         return true;
       });
-      _nodesNeedingLayout.add(this);
+      if (owner != null)
```

Comment:
If you mean ```owner?._nodesNeedingLayout.add(this);```, that doesn't quite work because the second dereference might happen on null. I guess I could wrap the member access in a method instead.